source/set-non-nullable.d.ts

```diff
@@ -0,0 +1,36 @@
+import {Except} from './except';
+import {Simplify} from './simplify';
+
+/**
+Create a type removes nullish (optional, undefined and null) types from the given keys. The remaining keys are kept as is.
+
```

Comment:
Can you mention how it related to `SetRequired` and the inverse in the `SetRequired` doc comment?